ckanext/security/validators.py

```diff
@@ -1,17 +1,32 @@
 # encoding: utf-8
-import six
 import string
 
+import six
```

Comment:
`six` is a good library, but...is this plugin actually compatible with Python 2? If not, then there's no point.

Reply:
> git blame says you introduced it 2021

Sure. In 2021, CKAN 2.8, and thus Python 2, were still current.

Reply:
I just wondered where six came from, that's why I mentioned it. :) Please don't take that as criticism.
I just open an issue to track this so it can be done in another PR.